playwright/_impl/_browser_type.py

```diff
@@ -166,6 +167,8 @@ async def connect_over_cdp(
         headers: Dict[str, str] = None,
     ) -> Browser:
         params = locals_to_params(locals())
+        if params.get("headers"):
```

Comment:
Will that be applied to the sync context as well?

Reply:
yes